### DISCUSSION TITLE
Land Cover / Railways / Time of day traffic

### DISCUSSION THREAD

**Discussion by arcticf** [Universite-Gustave-Eiffel/NoiseModelling#544] [Q&A] [answered]
Hi everyone, 

I had a few questions that I couldn't find an answer to.

1. I have seen that ground absorption could be used as an input table for noise map modelling. However, I was wondering if we could use land cover or even just "open" (e.g. pastures) and "closed" (e.g. forests) areas as an input too ? As ground absorption doesn't account for obstacles like trees.

2. I was following the OSM guide but I could only find roads and buildings, is there data for railways in the .osm.pbf file or do I need to add it separately ?

3. Finally, is there a possibility to distinguish hourly or day/night traffic using OSM data ? (and thus accounting for heavy going to/return from work traffic) 

Thanks in advance for your help !

**Comment by pierromond** [comment, marked as answer]
Hi @arcticf ,

1. In CNOSSOS-EU only the ground is taken into account (0<G<1). The effect of foliage is considered minor.
2. Nope, we can't import railways using OSM import. If road traffic is already very difficult to estimate/predict, rail traffic is nearly impossible. These data are not (yet) available on OSM.
3. Yes, day, evening and night are three pre-define periods.

**Reply by arcticf** [reply]
Hi @pierromond,

Thank you for your answer and time.
I will definitely have a look into the pre-defined periods.
Do you have any recommandations on where/how I can find railways (and traffic) for France to import ?

Thanks in advance.

**Reply by pierromond** [reply]
Railways, definitely SNCF, but hard to get. 
Roads, small ones are own by the different agglomerations, you need to contact them one by one. Biggest one are most of the time published on data.gouv.fr for example.
